### ISSUE TITLE
wallet: feedback missing on a failed operation in headless mode

### ISSUE THREAD

**Issue by t00ts** [dusk-network/rusk#643]
**Describe the bug**
We're not getting feedback for a failed transaction in headless mode:

**To Reproduce**
```
⚡ rusk-wallet address --key 1
Using config from /home/xxx/.dusk/config.toml
✔ Please enter wallet password: · [hidden]
> 3XYuDqAwD95ah7QKQSj44VTDxKyQhXBHxFJeg5qHHcC4rKm7nhtTk9FsTtG8rnpB4KnhwAFJ8ethBaXdZdr5K6Fo
```

```
⚡ rusk-wallet transfer --key 0 --rcvr 3XYuDqAwD95ah7QKQSj44VTDxKyQhXBHxFJeg5qHHcC4rKm7nhtTk9FsTtG8rnpB4KnhwAFJ8ethBaXdZdr5K6Fo -a 100 --gas-limit 10 --gas-price 1
Using config from /home/xxx/.dusk/config.toml
✔ Please enter wallet password: · [hidden]
0bc450569fe07bc8e195f0fae3da7727b953b5bb3d4d093f530e8d4cb055650a
```

```
⚡ rusk-wallet balance -k 1
Using config from /home/xxx/.dusk/config.toml
✔ Please enter wallet password: · [hidden]
> Total balance for key 1 is: 0 Dusk
> Maximum spendable per TX is: 0 Dusk
```

**Expected behaviour**
An error or any kind of feedback

**Comment by herr-seppia** [comment]
This tx has been successfully propagated to the network. And it has been executed too. (you can look at it in the explorer)
Wallet side, there is no error.

Maybe this can lead to a FeatureRequest like "Wait for tx confirmation" or something similar

**Comment by t00ts** [comment]
Inherently fixed by #683
